src/Bonsai.ML.Torch/Linspace.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.ComponentModel;
+using System.Reactive.Linq;
+using static TorchSharp.torch;
+
+namespace Bonsai.ML.Torch
+{
+    /// <summary>
+    /// Creates a 1-D tensor of linearly interpolated values within a given range given the start, end, and count.
+    /// </summary>
+    [Combinator]
+    [Description("Creates a 1-D tensor of linearly interpolated values within a given range given the start, end, and count.")]
+    [WorkflowElementCategory(ElementCategory.Source)]
+    public class Linspace
```

Comment:
Uff, this is another really hard one. `linspace` is really a horrible name ever since its inception, and the same as `Arange` really weird to PascalCase correctly. It is not even clear to me whether it should be `Linspace` or `LinSpace`.

If we were to depart from their convention, it might be better to call it `LinearSpace`, or `LinearRange`.